docs/source/getting_started.rst

```diff
@@ -39,7 +39,9 @@ Then, build their code with the following:
 
 .. code-block:: sh
 
-    mkdir build && cd build cmake .. make simulator-cli
+    mkdir build
+    cd build cmake 
```

Comment:
this line is still wrong, no? double check the ER-force instructions, but pretty sure it should be this:

```suggestion
    cd build/
    cmake ..
```